deploy.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+set -e # Exit with nonzero exit code if anything fails
+
+TARGET_BRANCH="gh-pages"
+
+function doCompile {
+  GRADLE_OPTS='-Xmx600m -Dorg.gradle.jvmargs="-Xmx1500m"' ./gradlew release
```

Comment:
As the `after_success` job is run independently of the build steps (see my comment before), I need to build it here, as I cannot garantee the build that runs before the deploy script.